pyproject.toml

```diff
@@ -80,3 +80,5 @@ test-unit = "pdm run make test-unit"
 test-integration = "pdm run make test-integration"
 test-e2e = "pdm run make test-e2e"
 coverage-report = "pdm run make coverage-report"
+check-types = "pdm run make check-types"
+schema-types = "pdm run make schema"
```

Comment:
More like `generate-schema` instead of `schema-types`?

Reply:
yup will be better

Reply:
updated @onmete